src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -206,7 +206,7 @@ class HeaderHelpMenuUI extends Component<Props, State> {
         <EuiButtonEmpty href={opensearchDashboardsDocLink} target="_blank" size="xs" flush="left">
           <FormattedMessage
             id="core.ui.chrome.headerGlobalNav.helpMenuOpenSearchDashboardsDocumentationTitle"
-            defaultMessage="OpenSearch Dashboards documentation"
+            defaultMessage="Wazuh documentation"
```

Comment:
Maybe port to #18

Reply:
The change took the other Issue https://github.com/wazuh/wazuh-dashboard/issues/18